.ci/azure-pipelines/build/macos.yaml

```diff
@@ -3,8 +3,7 @@ steps:
     # find the commit hash on a quick non-forced update too
     fetchDepth: 10
   - script: |
-      brew install pkg-config qt5 libpcap brewsci/science/openni libomp
-      brew install vtk --with-qt --without-python@2
+      brew install glew pkg-config qt5 libpcap brewsci/science/openni libomp
```

Comment:
glew was added explicitly here.

Reply:
2 questions:
* Why remove `brew install vtk`
* Why not delegate everything to brew?

(Wouldn't that make it a chicken egg problem?)

Reply:
It is installed automatically in
```
brew install --only-dependencies pcl
```

I had it separately before because we needed specific options.

Reply:
My sole point (maybe for a separate PR?) is we should do one of 2 things:
* Delegate everything to `brew install --only-dependencies pcl`
* Manually specify the dependencies

But not both

Reply:
The first one is definitely not possible, because that relies on the idea that the dependency list of homebrew pcl is always up to date with respect to dependencies pcl uses and that is not the case. We define what is needed, homebrew later updates.

This is currently what homebrew defines:
https://github.com/Homebrew/homebrew-core/blob/09ce3632b4d09e3a2ac4eb19dda4532a1dd768c1/Formula/pcl.rb#L16-L25

And we have this extra which is not reported in the build formula:
* glew 
* libpcap
* brewsci/science/openni
* libomp

So your first proposal is the way to go.